docs/data/date-pickers/date-picker/examplesConfig.ts

```diff
@@ -129,9 +129,11 @@ export const datePickerExamples: PickersSubcomponentType = {
         type: 'success',
       },
       sxProp: {
-        type: 'success',
+        type: 'warning',
+        parentSlot: 'desktopPaper',
```

Comment:
WDYT about using the `layout` slot instead? 🤔 
That slot is available for all the picker variants and wraps all the elements.
Suggesting the usage of the `desktopPaper` slot limits the application of the given style to only the `Desktop` picker.

Reply:
Makes sense 👍